### PR TITLE
doc(xresources): add examples for opacity and disableAlphaCorrection

### PR DESCRIPTION
**Pull request by actionless**
not sure about the wording, also my english is quite bad

### CODE REVIEW

File: .Xresources [hunk 1]
```diff
@@ -57,6 +57,15 @@ st.boxdraw_bold: 0
...
 st.boxdraw_braille: 0
 
+! opacity==255 means what terminal will be not transparent, 0 - fully transparent
+! (float values in range 0 to 1.0 could be used instead)
+st.opacity:      255
+! (0|1) don't pre-multiply color values of the pixels when using transparency
+! (useful for the systems where composite manager not running all the time,
+! as color pre-multiplication will make the "opaque" colors look darker/lighter
+! without the composition)
+st.disableAlphaCorrection: 0
```
Comment (actionless) [inline .Xresources:67]: also it's a bit bothering me what in xresources we have keynames like `st.foobar`, `st.foo_bar` and `st.fooBar`

Comment (neeasade) [reply .Xresources:67]: sameee. I think disableAlphaCorrection should maybe be changed to snake_case, and then maybe we consider moving the example .Xresources file to the wiki

**Comment by actionless** [comment]
I think is better for the file to stay together with the code

File: .Xresources (hunk 1 above)
Comment (actionless) [reply .Xresources]: now i'm on my pc, pushed the change here

**Pull request merged**